<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton {
	background-color: #39424f;
	border: 1px solid #39424f;
	border-radius: 4px;
	color: #fafafa;

}
QPushButton:hover {
	background-color: #BBC7DA;
}
QPushButton:focus {
	outline: none;
	border: 1px solid #8699B5;
	text-decoration: underline;
}
QPushButton:pressed {
	background-color: #8699B5;
}

QListWidget {
	border: none;
}
QListWidget QWidget {
	margin: 0;
	padding: 0;
}

QListWidget QPushButton {
	width: 16px;
	height: 16px;
	margin: 0;
	background-color: #fff;
	border: none;
	color: orange;
}
QListWidget QPushButton:hover {
	background-color: orange;
	color: white;
}

QTextEdit {
		border: 1px solid #8699B5;
	border-radius: 4px;
}
QTextEdit:focus {
	border: 3px solid #8699B5;
}

QListView::item:alternate {
    background: #EEEEEE;
}

QListView::item:selected {
    border: 1px solid #8699B5;
	color: #000;
	font-weight: 800;
	background-color: #fff;
}

QListView::item:selected:!active {
    border: 1px solid lightgrey;
}

QListView::item:hover {
    border: 1px solid lightgrey;
}

QLineEdit {
	border: 1px solid #66BB6A;
	border-radius: 4px;
}

QLineEdit[echoMode=&quot;2&quot;] {
    lineedit-password-character: 9679;
}

QLineEdit:focus {
	border: 2px solid #8699B5;
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditChatName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetYourContacts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>161</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelChatName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter name of new chat</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetChatList">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>161</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelYourContacts">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Your contacts</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New chat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>31</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRemove">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>31</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
